vignettes/details.Rnw

```diff
@@ -16,6 +16,7 @@
 
 
 \begin{document}
+\SweaveOpts{concordance=TRUE}
 
```

Comment:
By the way, `devtools::build_vignettes()` will mimic the "build vignette" stage that occurs during installation (without installing the package).